playbooks/robusta_playbooks/pod_troubleshooting.py

```diff
@@ -312,20 +312,17 @@ def python_process_inspector(event: PodEvent, params: DebuggerParams):
         finding_type=FindingType.REPORT,
         failure=False,
     )
-    event.add_finding(finding)
 
     process_finder = ProcessFinder(pod, params, ProcessType.PYTHON)
     relevant_processes_pids = process_finder.get_pids()
     if not relevant_processes_pids:
         ERROR_MESSAGE = f"No relevant processes found for advanced debugging."
         logging.info(ERROR_MESSAGE)
         finding.add_enrichment([MarkdownBlock(ERROR_MESSAGE)])
-        return
-
-    finding.add_enrichment(
-        [MarkdownBlock(f"Please select an advanced debugging choice:")]
-    )
-    if params.interactive:
+    elif params.interactive:
```

Comment:
I'm changing it. This code is too obfuscated.

It does hurt the user, in case he's using a sink that doesn't support callbacks.
he will see just:
"Please select an advanced debugging choice:"
"*After clicking a button please wait up to 120 seconds for a response*"

The choices clocks aren't there